analytics/crowd/mqtt2db.py

```diff
@@ -58,7 +58,7 @@ def on_message(self, client, userdata, message):
             r["time"]=int((r["real_base"]+r["timestamp"])/1000000)
 
             if "objects" in r and scenario == "traffic": r["nobjects"]=int(len(r["objects"]))
-            if "objects" in r and scenario == "stadium": r["count"]={"queue":len(r["objects"])}
+            if "objects" in r and scenario == "stadium": r["count"]={"crowd":len(r["objects"])}
```

Comment:
There is no count.crowd in the UI code. It's always count.people, regardless of the mode.

Reply:
Improved.